src/common/mathlib.F

```diff
@@ -1736,8 +1727,11 @@ ELEMENTAL SUBROUTINE eval_transc_func(r, eps, omega, fn, df)
       REAL(dp), INTENT(in)                               :: r, eps, omega
       REAL(dp), INTENT(out)                              :: fn, df
 
-         fn = erfc(omega*r) - r*eps
-         df = -omega*2*EXP(-omega**2*r**2)/SQRT(pi) - eps
+      REAL(dp)                                           :: qr
+
+         qr = omega*r
+         fn = ERFC(qr) - r*eps
+         df = -2.0_dp*oorootpi*omega*EXP(qr**2) - eps
```

Comment:
It looks like a minus sign before `qr**2` has been lost here.

Reply:
Hmm, I thought the minus took precedence like negative number squared?

( Previous expression was `df = -omega*2*EXP(-omega**2*r**2)/SQRT(pi) - eps`. )